src/components/sidenav/Sidenav.jsx

```diff
@@ -120,7 +120,7 @@ const Sidenav = (props) => {
           </Link>
         </div>
         {isUpdateAvailable ? <React.Fragment>
```

Comment:
no need for ternary operator 
{isUpdateAvailable && ...updateDiv}
< div >
< popover >...ui version</ popover >
...sc version
< /div >